internal/session/repository_session.go

```diff
@@ -289,11 +286,6 @@ func (r *Repository) ListSessions(ctx context.Context, opt ...Option) ([]*Sessio
 		where = append(where, fmt.Sprintf("s.public_id in (%s)", strings.Join(idsInClause, ",")))
 	}
 
-	if opts.withWorkerId != "" {
```

Comment:
Similar to my comments above, this is all sort of predicated on the idea that we will never want to query to find out which workers are _allowed_ for a given session. I'm not sure that's the case. It seems like information we might want to push into the warehouse with the other session information: the session contains the allowed set of workers, then each connection indicates which worker it actually used for that connection.